src/sematic-extra.css

```diff
@@ -23,4 +23,16 @@ button.tertiary {
 
 .ui.card.mentor-card:after {
   display: none;
-}
\ No newline at end of file
+}
+
+@media (min-width: 1025px) and (max-width: 1440px) {
```

Comment:
If any you can merge them by

```suggestion
@media (min-width: 1025px) and (max-width: 2560px) {
  .ui .cards {
    margin: 1rem;
  }
}
```

But again I wonder:
1. Why we need `margin: 1rem;` and not `margin: 0 1rem;`?
2. I think that `.ui .cards` is to much generic. Why not doing this on `.mentors-wrapper` which is our `className` (compare to `.ui` and `.cards` which comes with the framework) so we know exactly where this class is.